docs/migration-recipe.md

```diff
@@ -0,0 +1,106 @@
+Migrating to new host:
+----------------------
+
+### Prepare for migration:  
+
+1.  Ensure that salt-master, loadbalancer, and host in question are up, and that their health check for the relevant service is failing in haproxy after the host is fully up
+
+To view haproxy status: 
+
+-vagrant up the salt-master, loadbalancer, and host in question (`vagrant up`)
+
+-`vagrant ssh-config salt-master loadbalancer >> vagrant-ssh Ssh -L 4646:127:0.0.1:464 -F vagrant-ssh loadbalancer`
+
+-open loacalhost:4646/haproxy?stats in browser
+
+2.  Edit pillar data for roles.sls to include both old and new hostnames (ex. hostname*)
+
+### Migrate the host:
+
+#### Update Salt Master with latest config including prep from above
+
+1.  ssh into the salt-master server `ssh salt-master`
+
+2.  Navigate to srv/psf-salt `cd srv/psf-salt`
+
+3.  Run `git pull`
+
+4.  Run `sudo salt-call state.highstate`
+
+#### Ensure new configuration doesn't impact host being migrated
+
+1.  ssh into the host in question `ssh hostname`
+
+2.  Navigate to srv/psf-salt `cd srv/psf-salt`
```

Comment:
I don't think steps 2 and 3 here are relevant to hosts that aren't the salt sever, are they?